Lib/typing.py

```diff
@@ -991,6 +991,9 @@ def __subclasscheck__(self, cls):
         raise TypeError("Subscripted generics cannot be used with"
                         " class and instance checks")
 
+    def __dir__(self):
+        return list(set(object.__dir__(self)
```

Comment:
Shouldn’t this use a super() call? Or is there a reason that doesn’t work?

Reply:
Yup that will work. They both resolve to the same thing. I was just being paranoid (and also micro-optimizing :). I'll change it to super() for readability.